.github/workflows/docs.yml

```
@@ -0,0 +1,42 @@
+  cancel-in-progress: true
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup doc dependencies
+        run: |
+          pip install -f docs/requirements.txt
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)